docs/source/settings.rst

```diff
@@ -320,6 +330,24 @@ because it consumes less system resources.
    In order to use the inotify reloader, you must have the ``inotify``
    package installed.
 
+.. _reload-engine:
```

Comment:
`settings.rst` is auto generated from `config.py`, this means someone forget to generate them before.

https://github.com/benoitc/gunicorn/blob/master/CONTRIBUTING.md

> If you are adding a new configuration option or updating an existing one, please do it in gunicorn/config.py, then run make -C docs html to update docs/source/settings.rst.